config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
@jamesodhunt 

I agree with you that we should create container with different runtime in one single containerd, but I think you are using the wrong way.

If I understand correctly, config.json should only be interpreted by runtime but not containerd, if we put the runtime path or name in config.json, we need to handle it first in containerd then let container make decision of calling which runtime.

I think the correct way is making the `--runtime` and `--runtime-args` per-container flags but not containerd daemon flags, we don't need to remove all these two from daemon but leave them also as default configure. 

I think this is also what @crosbymichael 's saying " I don't think this should be part of the config" mean.  Actually I want to make per-container `--runtime` and `--runtime-args` long time ago, do you think it's acceptable or not? If OK, I can start working on this @crosbymichael . Sorry this is a little bit beyond the topic.